js/index.js

```diff
@@ -1,23 +1,29 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span').innerText;
+  const quantity = product.querySelector('.quantity input').value;
+
+  let subTotal = price * quantity;
+
+  product.querySelector('.subtotal span').innerText = subTotal
+
+  return subTotal;
 }
 
 function calculateAll() {
-  // code in the following two lines is added just for testing purposes.
-  // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
-  // end of test
+  
+  let allProduct = document.querySelectorAll('.product')
```

Comment:
- Is the empty line 16 useful? If not, it can be removed.
- The variable allProduct can be a const.